### PR TITLE
Storybook setup

### PR DESCRIPTION
**Pull request by pgdureza**

### CODE REVIEW

File: apps/storybook/package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "storybook",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "concurrently \"yarn:watch:*\"",
```
Comment (jubalm) [inline apps/storybook/package.json:7]: this needs yarn installed? will we be able to reuse pnpm if that's what turbo is set up with?

**Review by jubalm — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
   "scripts": {
     "build": "turbo run build",
     "dev": "turbo run dev --parallel",
+    "sb": "cd apps/storybook && pnpm dev",
```
[the remark this answers is not in the document]
Comment (pgdureza) [reply package.json:12]: maybe can compromise? I don't want to install turbo cli globally. so maybe we can have turbo.json entry and still have sb command here that just runs `turbo run storybook` instead.

File: apps/storybook/package.json (hunk 1 above)
Comment (pgdureza) [reply apps/storybook/package.json:7]: true. will update to pnpm.

File: package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (rjvillanueva89) [reply package.json]: if it's for local only; we could skip adding the command to turbo

**Review by jubalm — APPROVED** [review]

**Pull request merged**